templates/microprofile-payara/files/README.md

```diff
@@ -1,14 +1,22 @@
-# Payara Micro Maven template for Platform.sh
+# Payara Micro Maven for Platform.sh
 
-This project provides a starter kit for Payara Micro projects hosted on [Platform.sh](http://platform.sh).
+This project provides a starter kit for Payara Micro projects hosted on Platform.sh.
 
-We include recommended .platform.app.yaml and .platform files that should suffice for most use cases. You are free to tweak them as needed for your particular site.
+Payara Micro is the open source, lightweight middleware platform of choice for containerized Java EE (Jakarta EE) microservices deployments.  Less than 70MB in size, Payara Micro requires no installation or configuration and no need for code rewrites  – so you can build and deploy a fully working app within minutes.
```

Comment:
This paragraph reads like marketing copy.  If we want to include a description of what the project being templated is, it should 1) Be used across all templates and 2) Be as neutral as possible, rather than markety.

Reply:
Additionally, we should then discuss if that's going to be in the template definition's description, too.  We've got the space for it if we want.  It would probably be more useful there than in the README so I'm open to it if others are.

Reply:
Thank you, I updated. Let me know what do you think.